src/parser.rs

```diff
@@ -6,20 +6,40 @@ use cow_rc_str::CowRcStr;
 use std::ops::Range;
 use std::ascii::AsciiExt;
 use std::ops::BitOr;
-use tokenizer::{self, Token, Tokenizer, SourceLocation};
+use tokenizer::{Token, Tokenizer, SourcePosition, SourceLocation};
 
 
 /// A capture of the internal state of a `Parser` (including the position within the input),
 /// obtained from the `Parser::position` method.
 ///
 /// Can be used with the `Parser::reset` method to restore that state.
 /// Should only be used with the `Parser` instance it came from.
-#[derive(PartialEq, Eq, Debug, Clone, Copy)]
-pub struct SourcePosition {
-    position: tokenizer::SourcePosition,
-    at_start_of: Option<BlockType>,
+#[derive(Debug, Clone)]
+pub struct ParserState {
+    pub(crate) position: usize,
```

Comment:
Yes, this raises the minimum Rust version for cssparser to 1.18. https://bugzilla.mozilla.org/show_bug.cgi?id=1383311 already made it 1.19 for Firefox.